account_draft_invoice_print/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Draft Invoice Print
```

Comment:
Set upper line and agpl target from as in new template
 https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst